src/ansible/roles/passkey/tasks/main.yml

```diff
@@ -18,3 +18,30 @@
       path: /opt/random.c
       state: absent
   when: passkey_support
+
+- name: Install vfido_ wrapper scripts into /opt/test_venv
+  copy:
+    src: vfido_*
+    dest: "{{ test_venv }}/bin"
+    owner: root
+    group: root
+    mode: 755
+
+- name: Install vfido systemd service unit file
+  copy:
+    src: vfido.service
+    dest: /etc/systemd/system
+
+- name: Start vfido service
+  systemd_service:
+    name: vfido
+    daemon_reload: true
+    state: started
+
+- name: Clone virtual-fido to test_venv
+  git:
+    repo: 'https://github.com/bulwarkid/virtual-fido.git'
```

Comment:
IMO we should pin a specific version to avoid any issues with the development process

Reply:
added version to use when cloning.